### DISCUSSION TITLE
Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230727

### DISCUSSION THREAD

**Discussion by brkirch** [brkirch/stable-diffusion-webui#26] [Announcements]
### Changelog

- Update the [generative-models](https://github.com/Stability-AI/generative-models) commit hash to Stability-AI/generative-models@45c443b316737a4ab6e40413d7794a7f5657c19f

<hr /><em>This discussion was created from the release <a href='https://github.com/brkirch/stable-diffusion-webui/releases/tag/'>Experimental Offline Standalone Mac Installer for Stable Diffusion Web UI (unofficial) 20230727</a>.</em>

**Comment by fxbeaulieu** [comment]
Wow this installer's optimizations are making SD XL work properly on my iMac M1 2021 with 16GB ! It runs way smoother (and so much faster) than on my Ubuntu laptop with a dedicated Nvidia RTX 3050 Ti GPU.

**Settings used for my first test** 
Euler with 20 sampling steps on the SD_XL_Base_1.0 safetensors model (with the sdxl-vae-fp16-fix VAE). Prompt size in the 150-225 tokens range and negative prompt in the 75-150 range. 

**Generation results**
20/20 [05:55<00:00, 18.75s/it]

Epic work and thanks a lot man 💯 🥇 

### Update
I was playing safe and running with --lowvram for the first test... this is even better than I thought ! 

I tried a generation while running with --medvram, settings are DPM++ 2M Karras for 20 steps and I added CodeFormer face restoration which I did not use in the first test. Total exec time : 04:13 !

**Comment by Lassinger** [comment]
Hi,
Deforum is still not working..?

many THX, Lassinger
